### ISSUE TITLE
I have exactly the same problem, even with everything updated. I can't solve it, if you can make a video or give tips please

### ISSUE THREAD

**Issue by remisco** [stackblitz-labs/bolt.diy#467]
I have exactly the same problem, even with everything updated. I can't solve it, if you can make a video or give tips please

_Originally posted by @jvxcampos in https://github.com/coleam00/bolt.new-any-llm/issues/403#issuecomment-2508432677_

**Comment by remisco** [comment]
Utilise l'api mistral
Mistral large

**Comment by dustinwloring1988** [comment]
Explain what you have done and your setup.

**Comment by dustinwloring1988** [comment]
Should be pretty simply if you are using docker only 2 steps really.

With my setup ( Windows 11, 4060 GPU, Node.js v18.20.4, Docker version 27.3.1, git version 2.47.0 )

1) Clone repo
2) Inside the repo run the following docker-compose --profile development up

**Comment by jvxcampos** [comment]
> Deve ser muito simples se você está usando docker apenas 2 passos reais.
> 
> Minha configuração (Windows 11, 4060 GPU, Node.js v18.20.4, Docker versão 27.3.1, git versão 2.47.0 )
> 
> 1. Clone Repo
> 2. Dentro do repositório execute o seguinte desenvolvimento docker-compose-profile

I'm running Ollama locally, my gpu is also a 4060 with 8gb vram, everything is running well and the code is being generated without problems. What happens is that the Preview screen goes blank and nothing appears at all.

**Comment by jvxcampos** [comment]
I'm not using docker

**Comment by jvxcampos** [comment]
https://github.com/user-attachments/assets/783f1601-6374-44af-98fb-ae7b0b89f0ed

**Comment by dustinwloring1988** [comment]
its hard to say ask it 'Make a tic tac toe game using only html, css and js' this will allow us to see if its package related or not. it should render a tic tac toe game for you to play as mine dose everytime locally

**Comment by dustinwloring1988** [comment]
https://youtu.be/4QhFiGblwr0

**Comment by jvxcampos** [comment]
https://github.com/user-attachments/assets/986ffead-01c4-4cbf-976c-2f762ce3c2be



> @jvxcampos its hard to say ask it 'Make a tic tac toe game using only html, css and js' this will allow us to see if its package related or not. it should render a tic tac toe game for you to play as mine dose everytime locally

**Comment by jvxcampos** [comment]
Seems to be a problem when using React

**Comment by dustinwloring1988** [comment]
please try the latest and let me know if you still have the problem

**Comment by jvxcampos** [comment]
> please try the latest and let me know if you still have the problem

Still having the same problem

**Comment by dustinwloring1988** [comment]
@jvxcampos please list you environment.  ( OS, how you are running this fork, how you are running ollama, any and all commands you ran after cloning it )

**Comment by jvxcampos** [comment]
Ryzen 5 3600
16gb ram
rtx 4060 8 vram
Running everything on a m2 kingston

Ollama is running locally. I'm not using Docker.

The only thing I did was remove Ollama's context limit, change it to env.local, change the DEFAULT_MODEL to Ollama's DEFAULT_MODEL in \app\utils\constants.ts, as there was an error when it tried to use the previously defined one.

In package.json I changed it to "dev": "remix vite:dev --host 0.0.0.0 --port 5173 --open" following guidance from a video.

> @jvxcampospor favor, liste seu ambiente. (OS, como você está executando este fork, como você está executando ollama, todos e quaisquer comandos que você executou após cloná-lo )

**Comment by jvxcampos** [comment]
> experimente apenas um conjunto de instalação de baunilha fresco o seguinte em .env.local:
> 
> ```shell
> OLLAMA_API_BASE_URL=http://localhost:11434
> DEFAULT_NUM_CTX=4096
> ```
> 
> ou tente
> 
> ```shell
> OLLAMA_API_BASE_URL=http://host.docker.internal:11434
> DEFAULT_NUM_CTX=2048
> ```

Wow, this greatly improved the speed at which the code was written. But still no results in Preview and a white screen

**Comment by jvxcampos** [comment]
> can you please do one more screen capture and once the white screen come up lkeave it for like 20 sec then go back to the code for me. followed by your terminal you launched it in. specify show me this in the web container at the end of your prompt


https://github.com/user-attachments/assets/335a425d-c422-46eb-8926-c209654577a1

**Comment by dustinwloring1988** [comment]
good news is it is not anything with bolt and ollama config. let me think about it and get back to you though maybe someone in the mean time can help as well

**Comment by jvxcampos** [comment]
> good news is it is not anything with bolt and ollama config. let me think about it and get back to you though maybe someone in the mean time can help as well

Yes, Ollama and bolt are generating the code very well and that really excites me! Thank you very much for your help, I'm sure we will have a solution soon

**Comment by kismonguz** [comment]
> 2024-11-30.18-44-42_c.mp4

Based on this video, your problem looks like similar than mine in this issue this: https://github.com/coleam00/bolt.new-any-llm/issues/400

**Comment by remisco** [comment]
Il faut utiliser mistral ai mistral large
Et prompt avec next.js
